runtime/src/iree/builtins/ukernel/README.md

```diff
@@ -1,155 +1,38 @@
-IREE Microkernels Library: `libukernel`
-=======================================
+# Microkernels library
 
-This library provides builtin microkernels to both the IREE VMVX module for
-runtime linkage and the IREE compiler for ahead-of-time compilation. Each
-deployment approach has tradeoffs and the intent with this library is to share
-the same compiler passes/infrastructure for emitting the microkernel ops and
-the same microkernel implementations.
+## Walk-through presentation
 
-## Runtime Linkage
+Here is a walk-through of how ukernels are built and used in IREE:
+https://gist.github.com/bjacob/2c160c102cee33562826c730945b49f2
```

Comment:
Should this be in this repo too? (its a nice doc and just worried that being in a gist means its less easy for others to contribute etc)

Reply:
I've been thinking about copying that gist into a blog post (https://iree.dev/community/blog/ ). That would still be pretty "frozen" to contributions compared to a developer doc though.

Reply:
If there's interest in making it a living document, it could go under one of the sections in https://iree.dev/developers/.

Reply:
Up to you :-)